1209/8472/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: OpenDeck
+owner: Shantea Controls
```

Comment:
This needs to be the same as the directory name of your org - 'shanteacontrols'.